pymongo/mongo_client.py

```diff
@@ -333,6 +332,11 @@ def __init__(
             a Unicode-related error occurs during BSON decoding that would
             otherwise raise :exc:`UnicodeDecodeError`. Valid options include
             'strict', 'replace', and 'ignore'. Defaults to 'strict'.
+          - Starting in version 4.0 you can pass a custom SRV service name by
+            using the ``srvServiceName`` option like so::
```

Comment:
This does not match how we document the other options. The line needs to start with:
```
- `<option name>`: description.
```

There's also no need to mention the `Starting in version 4.0` part.